PhotoPlusBackend/photoplus/.idea/misc.xml

```diff
@@ -7,7 +7,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="11" project-jdk-type="JavaSDK">
```

Comment:
w h y

Reply:
Just gitignore .idea folder already. Fixed in ea48623